packages/responder/src/types/playbookTypes.ts

```diff
@@ -25,6 +25,7 @@ export interface AlResponderAction {
     category ?: object;
     action: AlResponderActionLong;
     allowed_values?: object;
+    complemented ?: boolean; // just for ui, is to know if the action is completed with details or not
```

Comment:
seems this property name is not accurate, maybe isComplete would be more meaningful as a boolean

Reply:
thanks